src/vs/platform/environment/node/argv.ts

```diff
@@ -466,6 +466,31 @@ export function buildHelpMessage(productName: string, executableName: string, ve
 	return help.join('\n');
 }
 
-export function buildVersionMessage(version: string | undefined, commit: string | undefined): string {
```

Comment:
Nit: For ease of merging, I think it'd be better for us to have a `buildPositronVersionMessage` of our own and leave `buildVersionMesage` alone.